test/src/kbasesearchengine/test/parse/ObjectParserTest.java

```diff
@@ -0,0 +1,256 @@
+package kbasesearchengine.test.parse;
+
```

Comment:
What coverage does this test have for the class and its dependencies? For immediate dependencies I see:
SubObjectExtractor
SimpleSubObjectConsumer
IdMapper
SimpleSubObjectConsumer
ObjectJsonPath

Those may have further dependencies.
In some cases it may make more sense to refactor the class so the dependencies can be injected rather than embedded (especially for static method calls).

Reply:
Same here